gbif-occurrence-linkage/main.py

```diff
@@ -0,0 +1,161 @@
+import json
+import logging
+import os
+import uuid
+from datetime import datetime, timezone
+from typing import Dict
+
+import requests
+from kafka import KafkaConsumer, KafkaProducer
+
+logging.basicConfig(format='%(asctime)s - %(message)s', level=logging.INFO)
+
+
+def start_kafka() -> None:
+    """
+    Start a kafka listener and process the messages by unpacking the image.
+    When done it will republish the object, so it can be validated and storage by the processing service
+    """
+    consumer = KafkaConsumer(os.environ.get('KAFKA_CONSUMER_TOPIC'),
+                             group_id=os.environ.get('KAFKA_CONSUMER_GROUP'),
+                             bootstrap_servers=[os.environ.get('KAFKA_CONSUMER_HOST')],
+                             value_deserializer=lambda m: json.loads(m.decode('utf-8')),
+                             enable_auto_commit=True)
+    producer = KafkaProducer(bootstrap_servers=[os.environ.get('KAFKA_PRODUCER_HOST')],
+                             value_serializer=lambda m: json.dumps(m).encode('utf-8'))
+    for msg in consumer:
+        logging.info('Received message: ' + str(msg.value))
+        json_value = msg.value
+        specimen_data = json_value['digitalSpecimen']['data']
```

Comment:
This doesn't look right to me. Based on the backend, I think  json_value would be:

```
{ 
    "jobId":3456-6578-65789-12349,
    "object":{
       "DigitalSpecimen": {
           "ods:id":"...", 
           "ods:type":"...",
           "ods:topicOrigin":"..."
       }
    }
}            
```

where DigitalSpecimen is the serialized jsonschema2Po
(Basing the assumption on [this part](https://github.com/DiSSCo/dissco-core-backend/blob/9297fca9e0a4b800f54785718c5be667761e87f9/src/main/java/eu/dissco/backend/service/MachineAnnotationServiceService.java#L81) of the MAS service)

Reply:
You are right! Stupid missed this as I tested with the API response.